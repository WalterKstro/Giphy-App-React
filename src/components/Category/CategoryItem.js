const CategoryItem = ({url,title}) => {
    return (
        <div className="grid__item animate__animated animate__fadeInLeft">
            <figure>
                <img src={url} alt={title}/>
                <figcaption>
                    {title}
                </figcaption>
            </figure>
        </div>
    )
}

export default CategoryItem
