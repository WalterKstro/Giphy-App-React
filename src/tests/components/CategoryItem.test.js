import CategoryItem from '../../components/Category/CategoryItem'
import { shallow } from "enzyme";

describe('tests in componente <CategoryItem/>', () => {

    test('should show correctly the component <CategoryItem/>',() => {
        const renderedComponent = shallow(<CategoryItem/>)

        expect(renderedComponent).toMatchSnapshot()
    })

})