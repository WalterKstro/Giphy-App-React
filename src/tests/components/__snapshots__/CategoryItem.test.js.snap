// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in componente <CategoryItem/> should show correctly the component <CategoryItem/> 1`] = `
<div
  className="grid__item animate__animated animate__fadeInLeft"
>
  <figure>
    <img />
    <figcaption />
  </figure>
</div>
`;
